tests/pytest_framework/src/syslog_ng_config/statements/destinations/file_destination.py

```diff
@@ -57,6 +57,9 @@ def read_log(self):
     def read_logs(self, counter):
         return self.dd_read_logs(self.get_positional_option_value(), counter=counter)
 
+    def getPath(self):
+        return self.options[self.get_positional_option_name()]
```

Comment:
I think we should stick to Python conventions here, and use get_path() as the name of the function.